app/code/Magento/Newsletter/Controller/Subscriber.php

```diff
@@ -41,24 +49,67 @@ abstract class Subscriber extends \Magento\Framework\App\Action\Action
      */
     protected $_customerUrl;
 
+    /**
+     * @var ScopeConfigInterface
+     */
+    protected $_scopeConfig;
```

Comment:
Properties should be private if there are no reasons to make them protected. Also, by naming convention we don't use underscore in naming.

Reply:
@melnikovi - I have updated the variable name and updated the scope of possible variables.